src/Wizard/Platform/Readline/ReadlinePlatform.ts

```diff
@@ -0,0 +1,161 @@
+import { Platform } from '../Platform';
+import { TestType } from '../../Models';
+import { Tokens } from '../../';
+import { ConnectivityAnalyzer } from '../../Services';
+import { inject, injectable } from 'tsyringe';
+import readline from 'readline';
+import { URL } from 'url';
+
+@injectable()
+export class ReadlinePlatform implements Platform {
+  private rl: readline.Interface;
+  private readonly delimiter = '\n\r--\n';
```

Comment:
https://github.com/NeuraLegion/nexploit-cli/pull/135#discussion_r568049223

You should use system-specific EOL everywhere where it's possible